subworkflows/nf-neuro/preproc_t1/tests/main.nf.test

```diff
@@ -17,17 +17,17 @@ nextflow_workflow {
 
     tag "load_test_data"
 
-        setup {
-            run("LOAD_TEST_DATA", alias: "LOAD_DATA") {
-                script "../../load_test_data/main.nf"
-                process {
-                    """
-                    input[0] = Channel.from( [ "antsbet.zip" , "others.zip" ] )
-                    input[1] = "test.load-test-data"
-                    """
-                }
+    setup {
+        run("LOAD_TEST_DATA", alias: "LOAD_DATA") {
+            script "../../load_test_data/main.nf"
+            process {
+                """
+                input[0] = Channel.from( [ "antsbet.zip" , "others.zip" ] )
```

Comment:
It could be worth changing the T1 being processed for the lean T1 version (the one in T1w.zip). I changed it in my PR, and it reduced the test's length by a lot! Also, I think we could test `antsbet` only once in those tests, and use synthstrip for the remaining ones, it also speed up the tests quite significantly.